docs/source/user_guide/installation/install-from-source.rst

```diff
@@ -31,26 +31,27 @@ To install PyBaMM, you will need:
 - A C compiler (ex: ``gcc``).
 - A Fortran compiler (ex: ``gfortran``).
 - ``graphviz`` (optional), if you wish to build the documentation locally.
+- ``pandoc`` to run the example Jupyter notebooks.
```

Comment:
I'll suggest changing this sentence as follows, because it is currently required only because of the `nbsphinx` extension and when building the HTML documentation (we don't use it to build PDFs or if building other formats). The example Jupyter notebooks can be triggered without needing `pandoc`.

```suggestion
- ``pandoc`` (optional) to convert the example Jupyter notebooks when building the documentation.
```